packages/salesforcedx-vscode-apex/src/messages/i18n.ts

```diff
@@ -65,6 +65,8 @@ export const messages = {
     "Your project doesn't contain any Apex test methods. To run Apex tests, open a project that contains methods with @istest annotations or the testMethod keyword",
   java_runtime_missing_text:
     'Java runtime could not be located. Set one using the salesforcedx-vscode-apex.java.home VS Code setting. For more information, go to [Set Your Java Version](%s).',
+  java_runtime_local_text:
+    'Local Java runtime (%s) is unsupported. Set the salesforcedx-vscode-apex.java.home VS Code setting to a runtime outside of the current project. For more information, go to [Set Your Java Version](%s).',
```

Comment:
Could we also add a note in https://github.com/forcedotcom/salesforcedx-vscode/blob/develop/docs/_articles/en/getting-started/java-setup.md#advanced-setup - maybe along with a bit of explanation as to why this is disallowed?

Btw the https://github.com/forcedotcom/salesforcedx-vscode/blob/develop/docs/_articles/en/getting-started/images/apex-java-home-setting.png image linked from the above doc is missing. Can you please fix it. The page also seems to have a typo - `Officially Test`.

Reply:
Yes, I will add @sbudhirajadoc to the ticket as well, so we can get a doc change and review on this one.

Reply:
Created W-10375128 to track this

Reply:
@mohanraj-r sorry but can you please point me to the typo you found? Thanks so much.

Reply:
Yes thanks @sbudhirajadoc that was the typo